src/NUnitFramework/tests/Constraints/MsgUtilTests.cs

```diff
@@ -307,9 +315,9 @@ public static void EscapesNullControlChars()
         private const string S52 = "abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ";
 
         [TestCase(S52, 52, 0, S52, TestName = "NoClippingNeeded")]
-        [TestCase(S52, 29, 0, "abcdefghijklmnopqrstuvwxyz...", TestName = "ClipAtEnd")]
-        [TestCase(S52, 29, 26, "...ABCDEFGHIJKLMNOPQRSTUVWXYZ", TestName = "ClipAtStart")]
-        [TestCase(S52, 28, 26, "...ABCDEFGHIJKLMNOPQRSTUV...", TestName = "ClipAtStartAndEnd")]
+        [TestCase(S52, 26, 0, "abcdefghijklmnopqrstuvwxyz...", TestName = "ClipAtEnd")]
+        [TestCase(S52, 26, 26, "...ABCDEFGHIJKLMNOPQRSTUVWXYZ", TestName = "ClipAtStart")]
+        [TestCase(S52, 22, 26, "...ABCDEFGHIJKLMNOPQRSTUV...", TestName = "ClipAtStartAndEnd")]
```

Comment:
Is this related to you comment on the PR (pasted below)? If not then what is this change about?

> Whilst updating the documentation, I noticed an issue with the reporting when texts are too long.

Reply:
No, it isn't. The original code had one places determining if the actual visible string should be clipped and in the called method it actually reduced the `maxDisplayLength`. I moved all determination code into the first method `ClipWhenNeeded` and reduced the `ClipString` method to do only the clipping to the length passed in.
This means that the length passed into the `ClipString` method is 3 or 6 less than previously and hence the updated numbers in the test.